SPECS/clamav/clamav.spec

```diff
@@ -137,8 +139,8 @@ fi
 %dir %attr(-,clamav,clamav) %{_sharedstatedir}/clamav
 
 %changelog
-* Wed Apr 17 2024 Andrew Phelps <anphel@microsoft.com> - 0.105.2-5
-- Fix build break by adding BR for systemd
+* Thu Apr 18 2024 Betty Lakes <bettylakes@microsoft.com> - 1.3.1-1
```

Comment:
looks like part of the changelog was accidentally overwritten?

Reply:
Oh, right, weird